client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -24,255 +24,39 @@ import {
-	'.capital',
-	'.cards',
-	'.care',
-	'.careers',
-	'.cash',
-	'.casino',
-	'.catering',
-	'.center',
-	'.chat',
-	'.cheap',
-	'.church',
-	'.city',
-	'.claims',
-	'.cleaning',
-	'.clinic',
-	'.clothing',
-	'.coach',
-	'.codes',
-	'.coffee',
-	'.community',
-	'.company',
-	'.computer',
-	'.condos',
-	'.construction',
-	'.contractors',
-	'.cool',
-	'.coupons',
-	'.credit',
-	'.creditcard',
-	'.cruises',
-	'.dating',
-	'.deals',
-	'.delivery',
-	'.dental',
-	'.diamonds',
-	'.digital',
-	'.direct',
-	'.directory',
-	'.discount',
-	'.doctor',
-	'.dog',
-	'.domains',
-	'.education',
-	'.email',
-	'.energy',
-	'.engineering',
-	'.enterprises',
-	'.equipment',
-	'.estate',
-	'.events',
-	'.exchange',
-	'.expert',
-	'.exposed',
-	'.express',
-	'.fail',
-	'.farm',
-	'.finance',
-	'.financial',
-	'.fish',
-	'.fitness',
-	'.flights',
-	'.florist',
-	'.football',
-	'.foundation',
-	'.fr',
-	'.fund',
-	'.furniture',
-	'.fyi',
-	'.gallery',
-	'.gifts',
-	'.glass',
-	'.gmbh',
-	'.gold',
-	'.graphics',
-	'.gratis',
-	'.gripe',
-	'.guide',
-	'.guru',
-	'.healthcare',
-	'.hockey',
-	'.holdings',
-	'.holiday',
-	'.house',
-	'.immo',
-	'.in',
-	'.co.in',
-	'.net.in',
-	'.gen.in',
-	'.firm.in',
-	'.find.in',
-	'.org.in',
-	'.industries',
-	'.institute',
-	'.insure',
-	'.international',
-	'.investments',
-	'.jewelry',
-	'.kitchen',
-	'.land',
-	'.lease',
-	'.legal',
-	'.life',
-	'.lighting',
-	'.limited',
-	'.limo',
-	'.loans',
-	'.ltd',
-	'.maison',
-	'.management',
-	'.marketing',
-	'.mba',
-	'.media',
-	'.memorial',
-	'.partners',
-	'.parts',
-	'.photography',
-	'.photos',
-	'.pictures',
-	'.pizza',
-	'.place',
-	'.plumbing',
-	'.plus',
-	'.productions',
-	'.properties',
-	'.recipes',
-	'.reise',
-	'.reisen',
-	'.rentals',
-	'.repair',
-	'.report',
-	'.restaurant',
-	'.run',
-	'.salon',
-	'.sarl',
-	'.school',
-	'.schule',
-	'.services',
-	'.shoes',
-	'.shopping',
-	'.show',
-	'.singles',
-	'.soccer',
-	'.solar',
-	'.solutions',
-	'.supplies',
-	'.supply',
-	'.support',
-	'.surgery',
-	'.systems',
-	'.tax',
-	'.taxi',
-	'.team',
-	'.technology',
-	'.tennis',
-	'.theater',
-	'.tienda',
-	'.tips',
-	'.tires',
-	'.today',
-	'.tools',
-	'.tours',
-	'.town',
-	'.toys',
-	'.training',
-	'.university',
-	'.vacations',
-	'.ventures',
-	'.viajes',
-	'.villas',
-	'.vin',
-	'.vision',
-	'.voyage',
-	'.watch',
-	'.wine',
-	'.works',
-	'.world',
-	'.wtf',
-	'.zone',
-	'.fun',
-	'.host',
-	'.online',
-	'.press',
-	'.site',
-	'.space',
-	'.store',
-	'.tech',
-	'.website',
-	'.actor',
-	'.airforce',
-	'.army',
-	'.attorney',
-	'.auction',
-	'.band',
-	'.consulting',
-	'.dance',
-	'.degree',
-	'.democrat',
-	'.dentist',
-	'.family',
-	'.forsale',
-	'.futbol',
-	'.games',
-	'.gives',
-	'.haus',
-	'.immobilien',
-	'.kaufen',
-	'.lawyer',
-	'.engineer',
-	'.market',
-	'.moda',
-	'.mortgage',
-	'.navy',
-	'.news',
-	'.ninja',
-	'.pub',
-	'.rehab',
-	'.republican',
-	'.reviews',
-	'.rip',
-	'.rocks',
-	'.sale',
-	'.social',
-	'.software',
-	'.studio',
-	'.vet',
-	'.video',
+	'.beer',
```

Comment:
Could you put them in alphabetical order, just so that it's easier to scan?

Reply:
Yep....meant to do that. Thanks!